### DISCUSSION TITLE
Erase/Install installing macOS 13.6 instead of 13.5.1

### DISCUSSION THREAD

**Discussion by macinthass** [grahampugh/erase-install#431] [General]
Love the Erase/Install script. We use it conjunction with Nudge to install updates with a simple click. Works fantastic but I noticed with the most recent update that it install macOS 13.6 even though it is not officially out yet. We want to prevent that from happening in the future. How can we do this? Thanks!

**Comment by grahampugh** [comment]
Use 30.2 - this fixes it.

**Comment by macinthass** [comment]
Fantastic. Will do. Thanks for the speedy reply!
